src/components/expense.css

```diff
@@ -0,0 +1,21 @@
+#expense-table > div:first-of-type,
+#expense-table > div:last-of-type {
+    display: flex;
+    flex-direction: column;
+    width: 80%;
+}
+
+#expense-table > div > div {
+    display: flex;
+    flex-direction: row;
+}
+
+#expense-table > div > div:first-child {
+    background-color: #f2f2f2;
+}
+
+#expense-table div > div span:nth-child(n),
```

Comment:
passing `n` to `nth-child` is not valid. `nth-child` expects an index of an element to select by. So a valid example is:
`div span:nth-child(1)`
or
`div span:nth-child(2)`